### ISSUE TITLE
Test are falling with gmsh 4.9.X, where X>=0

### ISSUE THREAD

**Issue by carlosal1015** [inducer/gmsh_interop#21]
Hi, in the system Arch Linux the tests start falling since gmsh 4.9.0 or later, is working with gmsh 4.8.4 or earlier. The tests are for gmsh_interop 2021.1.
```py
======================================================================== test session starts =========================================================================
platform linux -- Python 3.10.1, pytest-6.2.5, py-1.11.0, pluggy-0.13.1
rootdir: /tmp/makepkg/python-gmsh_interop/src/gmsh_interop
collected 8 items                                                                                                                                                    

test/test_gmsh.py FFFFFFFF                                                                                                                                     [100%]

============================================================================== FAILURES ==============================================================================
_______________________________________________________________________ test_simplex_gmsh[1-2] _______________________________________________________________________

dim = 2, order = 1, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_simplex_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        mr = GmshMeshReceiverBase()
        source = ScriptSource(GMSH_SPHERE, "geo")
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd847311660>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
_______________________________________________________________________ test_simplex_gmsh[1-3] _______________________________________________________________________

dim = 3, order = 1, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_simplex_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        mr = GmshMeshReceiverBase()
        source = ScriptSource(GMSH_SPHERE, "geo")
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd845b03cd0>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
_______________________________________________________________________ test_simplex_gmsh[3-2] _______________________________________________________________________

dim = 2, order = 3, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_simplex_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        mr = GmshMeshReceiverBase()
        source = ScriptSource(GMSH_SPHERE, "geo")
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8459fe950>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
_______________________________________________________________________ test_simplex_gmsh[3-3] _______________________________________________________________________

dim = 3, order = 3, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_simplex_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        mr = GmshMeshReceiverBase()
        source = ScriptSource(GMSH_SPHERE, "geo")
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:118: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8459c0eb0>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
________________________________________________________________________ test_quad_gmsh[1-2] _________________________________________________________________________

dim = 2, order = 1, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_quad_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        if dim == 2:
            source = ScriptSource(GMSH_QUAD_SPHERE, "geo")
        else:
            source = ScriptSource(GMSH_QUAD_CUBE, "geo")
    
        mr = GmshMeshReceiverBase()
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:142: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8460f7af0>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
________________________________________________________________________ test_quad_gmsh[1-3] _________________________________________________________________________

dim = 3, order = 1, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_quad_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        if dim == 2:
            source = ScriptSource(GMSH_QUAD_SPHERE, "geo")
        else:
            source = ScriptSource(GMSH_QUAD_CUBE, "geo")
    
        mr = GmshMeshReceiverBase()
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:142: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8459fc070>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
________________________________________________________________________ test_quad_gmsh[3-2] _________________________________________________________________________

dim = 2, order = 3, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_quad_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        if dim == 2:
            source = ScriptSource(GMSH_QUAD_SPHERE, "geo")
        else:
            source = ScriptSource(GMSH_QUAD_CUBE, "geo")
    
        mr = GmshMeshReceiverBase()
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:142: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8459ebe20>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
________________________________________________________________________ test_quad_gmsh[3-3] _________________________________________________________________________

dim = 3, order = 3, visualize = False

    @pytest.mark.parametrize("dim", [2, 3])
    @pytest.mark.parametrize("order", [1, 3])
    def test_quad_gmsh(dim, order, visualize=False):
        if search_on_path(["gmsh"]) is None:
            pytest.skip("gmsh executable not found")
    
        if visualize:
            save_tmp_files_in = f"simplex_{order}_{dim}d"
        else:
            save_tmp_files_in = None
    
        from gmsh_interop.reader import generate_gmsh, GmshMeshReceiverBase
        from gmsh_interop.runner import ScriptSource
    
        if dim == 2:
            source = ScriptSource(GMSH_QUAD_SPHERE, "geo")
        else:
            source = ScriptSource(GMSH_QUAD_CUBE, "geo")
    
        mr = GmshMeshReceiverBase()
>       generate_gmsh(mr, source, dimensions=dim, order=order,
                save_tmp_files_in=save_tmp_files_in)

test/test_gmsh.py:142: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
gmsh_interop/reader.py:512: in generate_gmsh
    runner.__enter__()
gmsh_interop/runner.py:218: in __enter__
    if self.version < "4.0.0":
/usr/lib/python3.10/site-packages/pytools/__init__.py:704: in wrapper
    result = function(obj, *args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <gmsh_interop.runner.GmshRunner object at 0x7fd8460f6800>

    @property
    @memoize_method
    def version(self):
        from distutils.version import LooseVersion
        cmdline = [
                self.gmsh_executable,
                "-version"
                ]
    
        from pytools.prefork import call_capture_output
        retcode, stdout, stderr = call_capture_output(cmdline)
    
        # stderr can contain irregular info
        import re
>       version = re.search(r"[0-9]+.[0-9]+.[0-9]+", stderr.decode().strip()).group()
E       AttributeError: 'NoneType' object has no attribute 'group'

gmsh_interop/runner.py:174: AttributeError
========================================================================== warnings summary ==========================================================================
test/test_gmsh.py::test_simplex_gmsh[1-2]
test/test_gmsh.py::test_simplex_gmsh[1-3]
test/test_gmsh.py::test_simplex_gmsh[3-2]
test/test_gmsh.py::test_simplex_gmsh[3-3]
test/test_gmsh.py::test_quad_gmsh[1-2]
test/test_gmsh.py::test_quad_gmsh[1-3]
test/test_gmsh.py::test_quad_gmsh[3-2]
test/test_gmsh.py::test_quad_gmsh[3-3]
  /tmp/makepkg/python-gmsh_interop/src/gmsh_interop/gmsh_interop/reader.py:506: DeprecationWarning: Not specifying target_unit is deprecated. Set target_unit='MM' to retain prior behavior.
    runner = GmshRunner(source, dimensions, order=order,

test/test_gmsh.py::test_simplex_gmsh[1-2]
  /tmp/makepkg/python-gmsh_interop/src/gmsh_interop/gmsh_interop/runner.py:163: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
    from distutils.version import LooseVersion

-- Docs: https://docs.pytest.org/en/stable/warnings.html
====================================================================== short test summary info =======================================================================
FAILED test/test_gmsh.py::test_simplex_gmsh[1-2] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_simplex_gmsh[1-3] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_simplex_gmsh[3-2] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_simplex_gmsh[3-3] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_quad_gmsh[1-2] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_quad_gmsh[1-3] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_quad_gmsh[3-2] - AttributeError: 'NoneType' object has no attribute 'group'
FAILED test/test_gmsh.py::test_quad_gmsh[3-3] - AttributeError: 'NoneType' object has no attribute 'group'
=================================================================== 8 failed, 9 warnings in 26.70s ===================================================================
```

**Comment by inducer** [comment]
I think this might have already been fixed by #20. I've just released https://pypi.org/project/gmsh-interop/2021.1.1/ which should include this fix. Reopen/comment here if that doesn't address this issue.

cc @alexfikl

**Comment by carlosal1015** [comment]
Thank you, is [fixed](http://0x0.st/orjP.png).
